cts/CTStests.py

```diff
@@ -1412,6 +1412,22 @@ def __call__(self, node):
             tmpPats, self.Env["DeadTime"] + self.Env["StableTime"] + self.Env["StartTime"])
         watch.setwatch()
 
+        if self.CM.Env["have_systemd"]:
+            temporary_corosync_service = """
+[Unit]
+Description=Temporary Corosync Cluster Engine to delay restarting the service
+
+[Service]
+ExecStart=/usr/share/corosync/corosync start
+ExecStop=/usr/share/corosync/corosync stop
+Type=forking
+Restart=always
```

Comment:
In order to prevent corosync from being recovered so that node gets fenced, shouldn't we need "Restart=no" in here?